packages/app/src/components/LikeButton.jsx

```diff
@@ -45,19 +61,26 @@ class LikeButton extends React.Component {
           id="like-button"
           onClick={this.handleClick}
           className={`btn btn-like border-0
-          ${pageContainer.state.isLiked ? 'active' : ''} ${isGuestUser ? 'disabled' : ''}`}
+            ${isLiked ? 'active' : ''} ${isGuestUser ? 'disabled' : ''}`}
         >
-          <i className="icon-like mr-3"></i>
-          <span className="total-likes">
-            {pageContainer.state.sumOfLikers}
-          </span>
+          <i className="icon-like"></i>
         </button>
-
         {isGuestUser && (
           <UncontrolledTooltip placement="top" target="like-button" fade={false}>
             {t('Not available for guest')}
           </UncontrolledTooltip>
         )}
+
+        <button type="button" id="po-total-likes" className="btn btn-like border-0 total-likes">
+          {sumOfLikers}
+        </button>
```

Comment:
Use [Button group](https://getbootstrap.com/docs/4.5/components/button-group/) please.

And I agree to rename this to `LikeButtons`.